_pages/robot/2018.html

```diff
@@ -45,12 +45,12 @@ <h5>Competition Performance</h5>
     <p>We have recently competed at two tournaments in the area. In our first competition at Battlefield High School, we ranked 2nd after qualifications and lead a strong alliance which sadly lost in the quarterfinals. In our second competition at Hayfield Secondary school, we were selected as members of the 2nd seed alliance and performed very well, making it to the semifinals before being eliminated. You can find out more about our competition performance on <a href="https://www.thebluealliance.com/team/1418">our Blue Alliance page</a>, and look out for videos about our competitions on our <a href="https://www.youtube.com/user/Robotics1418">YouTube page</a>.</p>
     <div class="iframe-ext-wrapper">
         <div class="iframe-wrapper">
-            <iframe width="560" height="315" src="https://www.youtube.com/embed/njAKg-UoMkA" allowfullscreen></iframe>
+            <iframe width="560" height="315" src="https://youtu.be/BqPwZ-Kcglw" allowfullscreen></iframe>
```

Comment:
Pretty sure this won't work. You need to use an embed, not just the link to the video.